src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -266,15 +274,24 @@ def reinit_data(self):
         """
         # establish connection to application database.
         self.db_conn.connect(mibs.APPL_DB)
-        self.mgmt_ip_str = self.db_conn.get(mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE,
-                                            b'lldp_loc_man_addr').decode('utf-8')
+        mgmt_ip_bytes = self.db_conn.get(mibs.APPL_DB, mibs.LOC_CHASSIS_TABLE, b'lldp_loc_man_addr')
+
+        if not mgmt_ip_bytes:
```

Comment:
>mgmt_ip_bytes [](start = 15, length = 13)

I believe it is already in another PR. Please don't mix. Any PR should based on current master.

Reply:
Rebased on latest master.